doi/src/main/java/ca/nrc/cadc/doi/ServiceAvailability.java

```diff
@@ -156,9 +165,13 @@ public Availability getStatus() {
                 log.debug("not configured: " + Standards.UMS_USERS_10);
             }
 
-            URI groupsURI = null;
+            URI groupsURI;
             try {
-                groupsURI = localAuthority.getServiceURI(Standards.GMS_SEARCH_10.toString());
+                Set<URI> groupsURIs = localAuthority.getServiceURIs(Standards.GMS_SEARCH_10);
```

Comment:
Not sure why you need the GMS configured in the `doi.properties` when you use the one in `cadc-registry.properties`. Am I missing something?

Reply:
Removed the GMS configuration, no longer needed. Removed the int test properties file, and added a intTest README to explain the int test setup.